Misc/NEWS.d/next/Library/2018-12-09-19-20-12.bpo-28053.JiOeSY.rst

```diff
@@ -0,0 +1 @@
+Fix the implementation of custom reducers for in `multiprocessing`.
```

Comment:
'for in' Either a word is missing or there is an extra word.

Reply:
To me, it looks like "for" is the extra word. I'd personally go with:

```suggestion
Fix the implementation of custom reducers in :mod:`multiprocessing`.
```

(Also adds the sphinx role `:mod:` to provide an inline link to the module. It's optional for `Misc/NEWS.d`, but we may as well include it with the typo fix. This makes it easier to move over to the "What's New" doc later.)

Reply:
Also note that `` `multiprocessing` `` will cause `make check` to fail: it should either be `` :mod:`multiprocessing` `` (as suggested by @aeros)  or ` ``multiprocessing`` `.